.github/workflows/playwright.yml

```
@@ -0,0 +1,27 @@
+name: Playwright Tests
+on:
+  push:
+    branches: [ main, master ]
+  pull_request:
+    branches: [ main, master ]
```

We don't use `master` thus it's unneeded:

```suggestion
  push:
    branches: [ main ]
  pull_request:
    branches: [ main ]
```